src/components/controls/miscInfoText.js

```diff
@@ -27,23 +28,31 @@ export const AnimationOptionsInfo = (
   </>
 );
 
-export const PanelOptionsInfo = (
+export const PanelLayoutInfo = (
   <>
-    Control which panels are being displayed and whether to show the tree and the map side-by-side (<em>grid</em>) or expanded (<em>full</em>).
-    <br/>
-    Note that what options are available here are dataset specific!
+    Control whether to show the tree and the map side-by-side (<em>grid</em>) or expanded (<em>full</em>).
+  </>
+);
+
+export const EntropyInfo = (
+  <>
+    This panel displays the observed diversity across the current genome or a selected CDS. All options are in the panel itself.
   </>
 );
 
 export const FrequencyInfo = (
   <>
+    This panel displays the prevalence of a characteristic over time, determined by the <em>color-by</em>.
+    <br/>
     <em>Normalize frequencies</em> controls whether the vertical axis represents the entire dataset or only the samples currently visible (e.g. due to filtering).
     This option is not available when data is limited to prevent numerical issues.
   </>
 );
 
-export const MeasurementsOptionsInfo = (
+export const MeasurementsInfo = (
   <>
+    This panel displays measurements.
+    <br/>
```

Comment:
I've repurposed each panel options section to represent the panel as a whole. Naturally, this wants a short description of each panel. I've added most in 830b73de2d4d03f42b7272e8fd6f83e6616cfed1, but don't know what to write for the measurements panel.

@joverlee521 are you able to suggest a short description for this panel?

Reply:
Ah sure! How about:

```
This panel displays multidimensional data such as serological experimental results 
that are linked to samples in the tree.
```

Reply:
That's perfect, thank you!

Reply:
Actually, since one can toggle the tree off and keep measurements on, does it make sense to word it without assuming the tree is visible? Or should the behavior be changed such that measurements can only be shown when the tree is also shown?

Reply:
Hmm, the measurements can definitely be shown even if the tree is off. 

I guess what I'm trying to get across is that the panel cannot contain arbitrary data not related to at least one sample in the tree. But maybe that's obvious since they are all part of the same dataset?

Reply:
I think the text looks good as is. A (the?) central design of Auspice is that all data is derived from / linked to nodes in the tree.

Reply:
I see. I think your snippet is good! Could be read implicitly as
> … results that are linked to samples in the tree (which may be hidden).

Reply:
Added: 40ff36dfddf1771fbfdd3c48acc32938f43925de